client/src/helpers/APICalls/stripe.ts

```diff
@@ -0,0 +1,16 @@
+import { StripeData } from '../../interface/StripeData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const stripe = async (): Promise<StripeData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    credentials: 'include',
+  };
+  return await fetch(`/connect/stripe`, fetchOptions)
+    .then((res) => res.json())
+    .catch(() => ({
+      error: { message: 'Unable to connect to server. Please try again' },
+    }));
+};
```

Comment:
Using `then()`/`catch()` makes `async`/await` redundant. Please use `async`/`await` properly.

Reply:
Currently every file in the ```APICalls``` folder has this same redundancy. I'll fix it in ```APICalls/stripe.ts``` for this PR, but the others should be brought up in their own issue so the rest of the app can be tested again.